deploy/osd-project-request-template/02-clusterrole.dedicated-admins-project-request.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Should this be a `Role` in the openshift-config namespace instead, as it refers to a single object?